### PR TITLE
Week 7 Email

### PR DESCRIPTION
**Pull request by jrtechs**
I created the announcement for week 7: graph databases. I also included a shout out to our new #news channel on slack.

### CODE REVIEW

File: announcements/_posts/2019-03-01-w7-graph-databases.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+date-end: "2019-03-01 18:00"
+location: "GOL-2650 (Lg. DB Lab)"
+---
+
+Hey RITluggers,
+
+This week we will be talking about graph databases and how they will change the way you think about data.
+If you are interested in big data and the oodles of open source software centered around it, be sure to check out this week's meeting.
+
+We now have a #news channel in our slack where we will be aggregating all the fun linux news of the week.
```
Comment (Tjzabel) [inline announcements/_posts/2019-03-01-w7-graph-databases.md]: `s/linux/Linux`

Also, in terms of news, the `#news` channel is simply supplementary. We were/are aiming to have a News category *in* the weekly emails we send out. That way, folks can stay up to date on Linux news without coming to our meetings.

Essentially, I would like the new format of our weekly newsletters/announcements to be:
```
INTRO
---------------
Hey RITluggers! (.....)


NEWS
---------------
Current Linux news for this week


UPCOMING EVENTS
---------------
RITlug events coming up in the near future
```

Comment (jrtechs) [reply announcements/_posts/2019-03-01-w7-graph-databases.md]: Do we have news for this week in the email?

Comment (ct-martin) [reply announcements/_posts/2019-03-01-w7-graph-databases.md]: Also, `s/slack/Slack`

Comment (Tjzabel) [reply announcements/_posts/2019-03-01-w7-graph-databases.md]: @jrtechs I was going to get some. Got caught up in some work. If you want to send the email out tonight, then just keep the email as it is. If you want to do it tomorrow early afternoon, then I can grab some news

Comment (jrtechs) [reply announcements/_posts/2019-03-01-w7-graph-databases.md]: @Tjzabel that would be great!

**Comment by Tjzabel** [comment]
## News

* [CentOS's 15th Birthday!](https://blog.centos.org/2019/02/centos15/)
* [KDE Plasma 15.5.2 Released!](https://kde.org/announcements/plasma-5.15.1-5.15.2-changelog.php)
* [34 New Members Join The Linux Foundation!](https://www.linuxfoundation.org/press-release/2019/02/34-new-members-join-the-linux-foundation-and-invest-in-open-source/)

**Comment by icflournoy** [comment]
2019-02-28-164207_858x646_scrot

**Comment by Tjzabel** [comment]
@icflournoy looks great!

**Pull request merged**